synapse/lib/storm.py

```diff
@@ -688,6 +690,47 @@ def cmpr(tufo):
 
         return cmpr
 
+    def _cmprCtorIval(self, oper):
+
+        name,tick = oper[1].get('valu')
+
+        minp = '>' + name
+        maxp = '<' + name
+
+        def cmpr(tufo):
+
+            minv = tufo[1].get(minp)
+            if minv is None or minv > tick:
+                return False
+
+            maxv = tufo[1].get(maxp)
+            if maxv is None or maxv <= tick:
+                return False
+
+            return True
+
+        return cmpr
+
+    def _cmprCtorIvalIval(self, oper):
```

Comment:
This function is untested.  Also, the use of s_interval would cause a NameError since synapse.lib.interval is not imported into storm.py.